test/fixtures/functionMsk/core.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+// NOTE: `kafkajs` is bundled into the deployment package
+// eslint-disable-next-line
```

Comment:
Let's indicate which exactly ESLint rule we're disabling (otherwise by accident we may shadow some other important eslint error)